locales/en/yourloops.json

```diff
@@ -122,6 +122,7 @@
   "avg-time-loop": "Avg. Daily Time In Loop Mode",
   "avg-time-loop-how-calculate": "**How we calculate this**:\n\n(%) is the duration in loop mode ON or OFF divided by the total duration of basals for this time period.\n\n**(time)** is the average daily time spent in loop mode ON or OFF.",
   "avg-time-loop-tooltip": "**Time In Loop Mode**: Daily average of the time spent in automated basal delivery.",
+  "below-range-tooltip": "Below Range",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏This one could simply be named `below-range`